Add files via upload

nikitz77 opened the pull request:
I could not add this through my Pycharm, so I am adding this through here

nikitz77 committed:
Add files via upload

I could not add this through my Pycharm, so I am adding this through here

Serg1oFG commented on HW1-Code.py:
```diff
@@ -0,0 +1,98 @@
+#1)Scatter plot
+iris_df.columns = ['Sepal Length','Sepal Width', 'Petal Length', 'Petal Width', 'Class']
+iris_df.head
+
```
Seems like you were listing down your plots, please continue adding them so the code looks more structured.

nikitz77 replied:
Thanks. Will do.

Serg1oFG commented on HW1-Code.py:
```diff
@@ -0,0 +1,98 @@
+
+
+
+#works but messy
```
Why is this messy? Might be good to explain why its messy, even if it still works.

nikitz77 replied:
Got it. Thanks. I am trying a trendline. It might go wrong. I will try this again tomorrow, but for now I am pushing some changes

Serg1oFG commented on HW1-Code.py:
```diff
@@ -0,0 +1,98 @@
+# and avoids leaking the test set into the train set
+pipe.fit(X_train, y_train)
+ 
+pipe.score(X_test, y_test)
```
Post some results of your observation as comments since they are not currently present.

nikitz77 replied:
Wow! I realized I uploaded the wrong file. This is outdated.

Serg1oFG reviewed (commented):
Comments left in-line with work. Overall looks good!

nikitz77 committed:
Update HW1-Code.py

Neater and has some preliminary results. I had also worked my train of thought here better.

Serg1oFG commented on HW1-Code.py:
```diff
@@ -20,79 +27,91 @@
-#works but messy
+##  3) Line graph
+#		group by class and have a different color for each class
+#       This looks messy because it was a line graph that had to go through each point in the scatterplot. I will have to refine this later to just reflect a trendline.
```
Awesome, this makes it easier to understand. I would just hit enter after the period (...scatterplot.) so that its not cut out from the viewer.

Serg1oFG reviewed (approved):
Just asked you to update a comment since it was cutting out, looks good after this to me!
